spec/event_spec.rb

```diff
@@ -0,0 +1,77 @@
+require './lib/person'
+require './lib/craft'
+require './lib/event'
+require 'pry'
+
+RSpec.describe Event do
+  it 'exists and has a name' do
+    person = Person.new({name: 'Hector', interests: ['sewing', 'millinery', 'drawing']})
+    craft = Craft.new('knitting', {yarn: 20, scissors: 1, knitting_needles: 2})
+    event = Event.new("Carla's Craft Connection", [craft], [person])
+
+    expect(event).to be_an_instance_of(Event)
+    expect(event.name).to eq("Carla's Craft Connection")
+  end
+
+  it "has start with no crafts and can crafts" do
+    person = Person.new({name: 'Hector', interests: ['sewing', 'millinery', 'drawing']})
```

Comment:
Make sure to really read the interaction pattern. `event.crafts` starts with an array of crafts. Same with `event.attendees`. You don't want to make more work for yourself.
```
pry(main)> event.crafts
#=> [#<Craft:0x00007fd37b818318...>]

pry(main)> event.attendees
#=> [#<Person:0x00007fd379b96028...>]
```